data_model/src/block.rs

```diff
@@ -383,9 +383,9 @@ mod candidate {
                 );
             };
 
-            if transactions.len() > 2 {
+            if transactions.len() > 3 {
                 return Err(
-                    "Genesis block must have one or two transactions (first with executor upgrade)",
+                    "Genesis block must have 1 to 3 transactions (executor upgrade, parameters, other isi)",
```

Comment:
you need to verify that all ISIs are `SetParameter` in the 2nd transaction

Reply:
if there is no parameters in genesis than 2nd transaction would be regular transaction